electron-builder.json

```diff
@@ -10,6 +10,8 @@
     ]
   },
   "linux": {"target": ["deb", "rpm"]},
+  "rpm": {"depends": ["openssl"]},
+  "deb": {"depends": ["openssl"]},
```

Comment:
FYI, this is the fix to #2485. Strictly speaking the RPM-based Fedora is the only place where I bumped into `openssl` being missing by default, therefore the addition of it as an explicit `deb` dependency as well is a bit "belt & suspenders". However, once I was aware this was an exposure in general I figured we might as well protect ourselves in case there's another Debian-based distro out there that similarly doesn't have it installed by default. FWIW I did test it out successfully on a Debian-based distro by manually uninstalling `openssl` then installing the Zui artifact and confirming that it required `openssl` to be reinstalled.